src/components/variable.jsx

```diff
@@ -43,275 +15,110 @@ const Variable = React.createClass({
+      foyer_fiscal: "un foyer fiscal",
+      individu: "un individu",
+      menage: "un ménage",
     }
-    if (formula.stop_instant) {
-      return (
-        <span>
-          Formule de calcul jusqu'au
-          {" "}
-          <FormattedDate value={formula.stop_instant} />
-        </span>
-      )
+    const definitionPeriodMessage = {
+      YEAR: "de l'année",
+      MONTH: "du mois",
+      ETERNITY: "de l'éternité (cette variable ne peut pas évoluer avec le temps",
     }
-  },
-  renderFormula(formula) {
-    const {parameters} = this.props
-    const inputVariableNames = formula.input_variables
-    const formulaParameterNames = formula.parameters
     return (
       <div>
+        <dl>
+          <dt>Cette variable est définie pour</dt>
+          <dd>{entityMessage[variable.entity]}.</dd>
+          <dt>Elle se calcule à l'échelle</dt>
+          <dd>{definitionPeriodMessage[variable.definitionPeriod]}</dd>
+          <dt>Elle est de type</dt>
+          <dd>{variable.valueType}</dd>
+          <dt>Sa valeur par défault est</dt>
+          <dd>{String(variable.defaultValue)}</dd>
         {
-          (inputVariableNames || formulaParameterNames) && (
-            <dl>
-              {inputVariableNames && <dt>Variables utilisées</dt>}
-              {
-                inputVariableNames && (
-                  <dd>
-                    <List items={sort(substract, inputVariableNames)} type="inline">
-                      {name => <Link to={`/${name}`}>{name}</Link>}
-                    </List>
-                  </dd>
-                )
-              }
-              {formulaParameterNames && <dt>Paramètres utilisés</dt>}
-              {
-                formulaParameterNames && (
-                  <dd>
-                    <List items={sort(substract, formulaParameterNames)} type="inline">
-                      {
-                        parameterName => {
-                          const parameter = parameters[parameterName]
-                          if (parameter) {
-                            return (
-                              <span>
-                                <Link title={parameter.description} to={`/${parameterName}`}>
-                                  {parameterName}
-                                </Link>
-                              </span>
-                            )
-                          } else {
-                            return (
-                              <span>
-                                {parameterName}
-                                {" "}
-                                <span className="label label-warning">inexistant</span>
-                              </span>
-                            )
-                          }
-                        }
-                      }
-                    </List>
-                  </dd>
-                )
-              }
-            </dl>
+          variable.references && (
+            <span>
```

Comment:
React imposes to have an "enclosing tag". We can get rid of it by un-factorizing the code I guess.